app/src/components/features/rules/RuleBuilder/Header/ActionButtons/CreateRuleButton/RuleInfoDialog/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useMemo } from "react";
+import { Col, notification, Row, Space } from "antd";
+//@ts-ignore
+import { CONSTANTS as GLOBAL_CONSTANTS } from "@requestly/requestly-core";
+//@ts-ignore
+import { ReactComponent as QuestionMarkIcon } from "assets/icons/question-mark.svg";
+import { RQButton } from "lib/design-system/components";
+import "./ruleInfoDialog.css";
+
+interface RuleInfoDialogContent {
+  title: string;
+  description: string;
+  readMoreLink?: string;
+  appMode?: string[];
+}
+
+const ruleInfoDialogContent: Record<string, RuleInfoDialogContent> = {
+  [GLOBAL_CONSTANTS.RULE_TYPES.RESPONSE]: {
+    title:
+      "Modified response body and status will not be visible in browser network dev tools.",
+    description:
+      "The rule will be executed correctly but cannot be monitored in dev tools due to technical limitions of the browser.",
+    readMoreLink:
+      "https://docs.requestly.io/browser-extension/chrome/http-modifications/response-rule",
+    appMode: [GLOBAL_CONSTANTS.APP_MODES.EXTENSION],
+  },
+  [GLOBAL_CONSTANTS.RULE_TYPES.REQUEST]: {
+    title:
+      "Modified request body will not be visible in browser network dev tools.",
+    description:
+      "The rule will be executed correctly but cannot be monitored in dev tools due to technical limitions of the browser.",
+    readMoreLink:
+      "https://docs.requestly.io/desktop-app/mac/http-modifications/request-body-rule",
+    appMode: [GLOBAL_CONSTANTS.APP_MODES.DESKTOP],
+  },
+};
+
+const ruleInfoDialog = (ruleType: string, appMode: string): void => {
+  if (!(ruleType in ruleInfoDialogContent)) return;
+  // if appMode is not present in <ruleInfoDialogContent> then show for all app modes
+  if (
+    ruleInfoDialogContent[ruleType].appMode?.length > 0 &&
+    !ruleInfoDialogContent[ruleType].appMode.includes(appMode)
+  )
+    return;
+
+  notification.open({
+    message: <MessageComponent ruleType={ruleType} />,
+    top: 60,
+    className: "rule-info-dialog-box",
+    duration: 0,
+  });
+};
```

Comment:
In designs we don't have any close icon in notification box
we can add `closeIcon` key here to hide the close icon